src/engine/V3Eval.v3

```diff
@@ -644,6 +646,11 @@ def I16_SUB_SAT_U(a: u16, b: u16) -> u16 {
 	if (dif < 0) return u16.view(0);
 	else return u16.view(dif);
 }
+def I16_Q15_MUL_SAT_S(a: i16, b: i16) -> i16 {
+	var prod = (i32.view(a) * i32.view(b) + 0x4000) >> 15;
+	if (prod > i16.max) return i16.max;
```

Comment:
You can turn this `if` into a one-liner with `return if(prod > i16.max, i16.max, i16.view(prod));`

Reply:
Yes that's a ternary statement here. Will push a change.

But I'm not sure why the CI failed. Will take a closer look tmw.

Reply:
The CI failed because of a breaking change. Should be fixed now.